metadata/index.html

```diff
@@ -538,47 +538,16 @@
-            <li>If the property is an <a>object property</a>, then if the property only accepts single objects:
-              <ol>
-                <li>if the value of the property in A is a string or the value from B is a string then the value from A overrides that from B</li>
-                <li>otherwise (if both values as objects) the objects are merged as described here</li>
-              </ol>
-              If the property accepts arrays, the result is an array of objects or strings: those from A followed by those from B that were not already a value in A.</li>
-            <li>If the property is a <a>natural language property</a>, the result is an object whose properties are language codes and where the values of those properties are arrays. The suitable language code for the values is either explicit within the existing value or determined through the <a>default language</a> in the metadata document; if it can't be determined the language code <code>und</code> should be used. The arrays should provide the values from A followed by those from B that were not already a value in A.</li>
-            <li>If the property is an <a>atomic property</a>, then if the property only accepts single values, the value from A overrides that from B; otherwise the result is an array of values: those from A followed by those from B that were not already a value in A.</li>
-            <li>If the property is <code>@context</code>, then the result is an array consisting of the URLs from A followed by the URLs from B not already in A followed by the merge of any object values of A and B.
-              If the resulting array contains only a single value, the result is that value.
-            </li>
-          </ul>
-          <p>
-            If the type of the property cannot be determined, because it is not defined in this specification (ie because it is an extension property), the type of the property is determined based on its values in A and B, as follows, and merged accordingly:
-          </p>
-          <ul>
-            <li>If the value of the property in A and the value of the property in B are both objects, they are treated as if the property is an <a>object property</a> that only accepts single objects.</li>
-            <li>If one of the values is an array and the other is an object, they are treated as if the property is an <a>object property</a> that accepts arrays.</li>
-            <li>If the value of the property in A and the value of the property in B are atomic values, they are treated as if the property is an <a>atomic property</a> that only accepts single values.</li>
-            <li>If one of the values is an array and the other is an atomic value, they are treated as if the property is an <a>atomic property</a> that accepts arrays.</li>
-          </ul>
-        </section>
+        <p>
+          A <code>@context</code> B is <a>merged</a> into an original <code>@context</code> A by:
```

Comment:
Obviously, this is part of a separate issue, but the consequence would be the same. If we resolve that way, then we can certainly simplify the merge language.